Stackdriver-Webhook/index.js

```diff
@@ -57,6 +56,59 @@ async function postToGoogleMonitoring(response) {
 		timeSeriesData[i] = parseTimeSeriesData(metric, dataPoint, testId, nodeName);
 	}
 
+	/** If Test type is Trace route then compute RTT, Packet loss, #Hops */
+	if (response.TestDetail.TypeId == 12) {
+	
+		let sumPingTime = 0;
+		let packetLossCounter = 0;
+		let pingCounter = 0;
+
+		let numberOfHops = response.Diagnostic.TraceRoute.Hops.length
+
+		for (var i = 0; i < 3; i++) {
+			if (response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i] != null) {
+
+				sumPingTime += response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i];
+				pingCounter++;
+			}
+			if (response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i] == null) {
+				packetLossCounter++;
+			}
+		}
+
+		let rtt = (sumPingTime / pingCounter).toFixed();
+		let packetloss = (33.333 * packetLossCounter).toFixed();
+
+                
+	        /** Datapoint for total number of hops */
+		let dataPoint = parseDataPoint(numberOfHops);
```

Comment:
Add `TotalHopsMetric`, `RoundTripTimeAverageMetric`, and `PacketLossPercentMetric` constants
```suggestion
		let dataPoint = parseDataPoint(numberOfHops);
		timeSeriesData.push(parseTimeSeriesData(TotalHopsMetric, dataPoint, testId, nodeName));

                /** Datapoint for  Round trip time */
		dataPoint = parseDataPoint(rtt);
		timeSeriesData.push(parseTimeSeriesData(RoundTripTimeAverageMetric, dataPoint, testId, nodeName));

                /** Datapoint for packet loss */
		dataPoint = parseDataPoint(packetloss);
		timeSeriesData.push(parseTimeSeriesData(PacketLossPercentMetric, dataPoint, testId, nodeName));
```